data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,773 @@
+			`	He chuckles as if having made a joke, as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`
+			`	"I am sure you have questions but they will have to wait until we're more established. We have supplies coming and they will be vulnerable to any pirates that may poke their noses in. We need you to get out there and ensure the supplies can all arrive here safely. Local security is still about, so it shouldn't be too problematic."`
+			`	He collects a device from a carry crate and hands it to you. "Now that we've got the basics up and running, the station is in safe mode. You'll need to plug this into your communications system to re-dock safely. The supply convoy has them pre-installed."`
+			choice
+				`	(Go immediately.)`
+					accept
+				`	(Loiter to find out more.)`
+					goto loiter
+				`	"What were we getting paid for this?"`
+			`	He frowns. "It was based on your ship's combat quality. I don't have that tablet with me right now though. Once the supply ships have arrived I should have access to that again. Now go on, you won't get paid at all if the supply ships don't make it."`
+				accept
+			label loiter
+			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
+			`	You are left with little choice other than to nod and get on your way.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+	on visit
+		dialog "The supply ships have not all arrived safely yet."
+	on fail
+		dialog "The supply convoy has been lost."
+	npc accompany
+		government "Hai Merchant"
+		personality escort timid
+		system "Ula Mon"
+		fleet "Large Hai Merchant" 3
+	npc
+		government "Pirate"
+		personality coward opportunistic vindictive
+		system "Waypoint"
+		fleet "Marauder fleet IX" 2
+	on complete
+		"reputation: Hai" += 200
+		"reputation: Hai (Wormhole Access)" += 200
+		"reputation: Hai Merchant" += 200
+		"reputation: Hai Merchant (Human)" += 200
+		"reputation: Hai Merchant (Sympathizers)" += 200
+		event "hr: sandsnap awoken" 5
+		fail "HRCP: Sandsnap Mercenaries"
+		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
+		payment 1200000
+		conversation
+			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
+			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`
+
 
+event "hr: sandsnap awoken"
+	planet Sandsnap
```

Comment:
Wouldn't it make more sense to use one of the mothballed stations currently in the game as a military base for HR, rather than add an entirely new one?

Reply:
Sandsnap is an existing station.

Reply:
Sorry, never mind then.

Reply:
No problem.